protos/network.proto

```diff
@@ -18,30 +18,20 @@ option java_multiple_files = true;
 option java_package = "sawtooth.sdk.protobuf";
 option go_package = "network_pb2";
 
-// The connect message from a client to the server
-message ConnectMessage {
```

Comment:
Do we lose some sort of a handshake by removing this message?  I.e. can a malicious process connect with  ZMQ sockets directly and cause memory issues by filling up the connections for a longer period of time?

Reply:
Based on in-person conversation, this has been replaced by ConnectionRequest in the authorization namespace of messages

Reply:
This message has been replaced by ConnectionRequest and is no longer used.